components/Player/Player.jsx

```diff
@@ -38,6 +38,7 @@ const Player = ({ playerId, info }) => {
     </div>
   );
 };
+// need to fix this
```

Comment:
I meant to remove this comment before pushing since it was just something I needed to resolve before pushing. Whoops